packages/client/src/__tests__/integration/happy/blog-env-cockroachdb/schema.prisma

```diff
@@ -0,0 +1,36 @@
+datasource db {
+  provider = "cockroachdb"
+  url      = env("TEST_COCKROACH_URI")
+}
+
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["Cockroachdb"]
+}
+
+// / User model comment
+model User {
+  id    String  @id @default(uuid())
+  email String  @unique
+  // / name comment
```

Comment:
Space inside the triple comment intended?

Reply:
I think it was copypasted from the other tests. We have a ton of tests with the same typo (maybe there used to be a bug in `prisma-fmt` in the past?).

Reply:
That comes from our other tests: 171 occurrences in 90 files. So something to fix but also mostly not needed as we only test comments in our type gen snapshots.